hoomd/wall.py

```diff
@@ -420,7 +427,7 @@ def _unsync(self):
         for wall_list in self._backend_lists.values():
             wall_list._unsync()
 
-    def _get_obj_backend_index(self, frontend_index, new_type, old_type=None):
+    def _get_0bj_backend_index(self, frontend_index, new_type, old_type=None):
```

Comment:
Why the name change?

Reply:
This was an errant find and replace. Will fix.